src/array_api_stubs/_draft/array_object.py

```diff
@@ -422,6 +426,10 @@ def __float__(self: array, /) -> float:
         - If ``self`` is ``True``, the result is ``1``.
         - If ``self`` is ``False``, the result is ``0``.
 
+        **Lazy implementations**
+
+        Lazy implementations of the API standard must raise a ``ValueError`` if they are unable eagerly compute the requested value.
```

Comment:
Could you change "must" to "should"? For why, see https://data-apis.org/array-api/draft/design_topics/exceptions.html